cloudflare/resource_cloudflare_device_posture_rule.go

```diff
@@ -0,0 +1,288 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceCloudflareDevicePostureRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareDevicePostureRuleCreate,
+		Read:   resourceCloudflareDevicePostureRuleRead,
+		Update: resourceCloudflareDevicePostureRuleUpdate,
+		Delete: resourceCloudflareDevicePostureRuleDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareDevicePostureRuleImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
```

Comment:
should this be required? it looks like we're accessing it directly further below for the API calls.

Reply:
Yep, good catch.